README.md

```diff
@@ -1 +1,56 @@
-# angular-scope-rootscope-readme
+# $scope and $rootScope
+
+## Overview
+
+## Objectives
+
+- Describe what a $scope is
+- Describe what $rootScope is
+- Describe how Angular utilises Prototypal Inheritance
+- Create a $scope variable
+- Bind the $scope variable to the View
+
+## $scope
+
+You might've noticed the magic of `$scope` in the last lesson - an object that we can assign values to and they appear in our view.
```

Comment:
it may be worth giving more of an overview of how $scope is working here. I see that we're passing it into our controller function - does it have to be called $scope? A non-technical description of what Angular is doing here will help give the reader a sense of context going forward.